androidHyperskillApp/src/main/java/org/hyperskill/app/android/study_plan/fragment/StudyPlanFragment.kt

```diff
@@ -44,8 +46,42 @@ class StudyPlanFragment :
     private var gamificationToolbarDelegate: GamificationToolbarDelegate? = null
     private var studyPlanWidgetDelegate: StudyPlanWidgetDelegate? = null
 
+    private val onForegroundObserver =
```

Comment:
Add activity into obsereverName for clearly show it is used with activity's lifecycle